astro/public/assets/css/base.css

```
@@ -154,4 +154,27 @@ Target Matrix
+  /* Bootstrap 5 theme colours (the site ships 5.0.2: --bs-success/-danger/-warning/-secondary/-info) */
+  --mark-score: #198754;
+  --mark-turnover: #dc3545;
+  --mark-flag: #ffc107;
+  --mark-stop: #6c757d;
+  --mark-explosive: #0dcaf0;
+}
+.play-marks { white-space: nowrap; margin-left: .5rem; font-size: 15px; line-height: 1; vertical-align: -1px; }
+.play-marks .pi + .pi, .play-marks .pi + .pi-pill, .play-marks .pi-pill + .pi, .play-marks .pi-pill + .pi-pill { margin-left: 4px; }
+/* SACK / TFL are text tags: fixed 14px so they sit inside the text line and never scale with the slot;
+   neutral outline (chosen 2026-09-01 over red/solid) in the stop tone. */
+.pi-pill { display: inline-block; box-sizing: border-box; vertical-align: 1px; height: 14px; padding: 0 4px; border: 1.5px solid var(--mark-stop); border-radius: 3px; color: var(--mark-stop); background: transparent; font: 700 9.5px/11px "Chivo", system-ui, sans-serif; letter-spacing: .06em; text-transform: uppercase; }
```

Stylelint (17.14.0)

[error] Expected no quotes around "Chivo" (font-family-name-quotes)

(font-family-name-quotes)